Sources/FluentDarkModeKitExample/ViewController.swift

```diff
@@ -17,8 +17,7 @@ class ViewController: UIViewController {
     )
   }
 
-  @objc
-  private func refresh() {
+  @objc private func refresh() {
```

Comment:
@kylinroc what do you think of this change?

Reply:
I personally have no strong preference.  But I like the new one.

Reply:
I'm ok with both style.